Resource widget (#1516)
kyma-project/busola: docs/extensibility/form-widgets.md

}
```

### Resource

> Resource widgets render a dropdown list of specified resources and stores the selected one as a string containing it's name.

Reviewer: ```suggestion
Resource widgets render a dropdown list of specified resources and store the selected one as a string containing its name.
```
- `stores` -> `store`: because `resource widgets` is plural
- `it's` = it is; `its` = belonging to **it**

#### Widget-specific parameters

- **kind** - _[required]_ Kubernetes kind of the resource.

> - **group** - API group used for all requests. Not provided for native kubernetes resources.

Reviewer: ```suggestion
- **group** - API group used for all requests. Not provided for native Kubernetes resources.
```

- **version** - _[required]_ API version used for all requests.

> - **scope** - either `namespace` or `cluster`. When set to cluster namespaced resources will be fetched from all namespaces. Defaults to `cluster`.

Reviewer: ```suggestion
- **scope** - either `namespace` or `cluster`. When set to `cluster`, namespaced resources are fetched from all Namespaces. Defaults to `cluster`.
```

> - **namespace** - namespace to fetch resources from. Used only when scope is `namespace` and resources need to be fetched from a specific namespace. Defaults to active namespace when omitted.

Reviewer: ```suggestion
- **namespace** - Namespace to fetch resources from. Used only when scope is `namespace` and resources need to be fetched from a specific Namespace. Defaults to the active Namespace when omitted.
```

> - **namespace** - namespace to fetch resources from. Used only when scope is `namespace` and resources need to be fetched from a specific namespace. Defaults to active namespace when omitted.

Reviewer: Consider making these parameters a table, like this:

```
| **Parameter** | **Description** |
|---------------|-----------------|
| **Example param** | Example description |
```

Author: Worth considering, but if so it would be better to change for all widgets - so I'd rather leave it as a separate documentation improvement task (only as an investigation, as I'm not actually sure if it's a good idea).
